stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/WebSocketObservabilityUtil.java

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+package org.ballerinalang.net.http.websocket;
+
+import org.ballerinalang.jvm.observability.ObservabilityConstants;
+import org.ballerinalang.jvm.observability.ObserveUtils;
+import org.ballerinalang.jvm.observability.ObserverContext;
+import org.ballerinalang.jvm.observability.metrics.DefaultMetricRegistry;
+import org.ballerinalang.jvm.observability.metrics.MetricId;
+import org.ballerinalang.jvm.observability.metrics.MetricRegistry;
+import org.ballerinalang.jvm.observability.metrics.Tag;
+import org.ballerinalang.jvm.observability.metrics.Tags;
+import org.ballerinalang.jvm.values.ObjectValue;
+import org.ballerinalang.net.http.websocket.server.WebSocketOpenConnectionInfo;
+import org.ballerinalang.net.http.websocket.server.WebSocketServerService;
+import org.ballerinalang.net.http.websocket.server.WebSocketService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Providing observability functionality to WebSockets.
+ *
+ * @since 1.1.0
+ */
+public class WebSocketObservabilityUtil {
+
+    private static final Logger logger = LoggerFactory.getLogger(WebSocketObservabilityUtil.class);
+
+    /**
+     * Observes incoming WebSocket connection requests.
+     *
+     * @param connectionInfo information regarding connection.
+     */
+    static void observeRequest(WebSocketOpenConnectionInfo connectionInfo) {
+        if (ObserveUtils.isObservabilityEnabled()) {
+            ObserverContext observerContext = initializeObserverContext(connectionInfo);
+            incrementCounterMetric(observerContext, WebSocketObservabilityConstants.METRIC_REQUESTS);
+
+            //Log request
+            logger.info("WS connection request received");
```

Comment:
This log gets printed in the console? Is it really neccassary? If it is, check if we can add details like connection ID etc.